### DISCUSSION TITLE
One wants his "resumé" with SILE :)

### DISCUSSION THREAD

**Discussion by Omikhleia** [sile-typesetter/sile#1272] [Show and tell]
Toying with tables, bullet lists and page masters to craft a CV sample.

![image](https://user-images.githubusercontent.com/18075640/140661187-5922920a-d3bc-401d-a2b8-893795e6abac.png)

**Comment by OlivierNicole** [comment]
Pretty nice, does it use a multi-page table?

**Reply by Omikhleia** [reply]
> Pretty nice, does it use a multi-page table?

Exactly, it builds a table programmatically out of the description provided in (hopefully) a simpler structure.

**Comment by alerque** [comment]
Pretty nice!

I'd be happy to include this on the website examples. I'm sure we can work out how to generate it there if the code is something you can share.

My only suggestion would be to use more vertical whitespace. I'd allocate *most* (not quite all) of the extra space at the end of page 2 to give the running header/footer and section titles more breathing room.

**Reply by Omikhleia** [reply]
> I'd be happy to include this on the website examples. I'm sure we can work out how to generate it there if the code is something you can share.

The code is currently in my omikhleia-sile-packages repository but besides the CV sample itself, it relies on a whole custom class, itself loading around 6-7 custom helper packages (e.g. style specifications, enumerations, tables, superscripts, cross-references, custom page headers), themselves with a few dependencies. I'm not sure how you'd prefer to proceed forward.

**Reply by alerque** [reply]
I'd like to proceed in such a way that our package dependency management is robust enough that even deep dependency trees like you describe are easy to setup for end users to take advantage of without having to tie their brains in knots getting there.

I really need to get back to work on #538.
